tests/foreman/ui/test_computeresource_vmware.py

```diff
@@ -361,6 +362,7 @@ def test_positive_vmware_custom_profile_end_to_end(
                 'provider_content.user': settings.vmware.username,
                 'provider_content.password': settings.vmware.password,
                 'provider_content.datacenter.value': settings.vmware.datacenter,
+                'provider_content.enable_caching': caching['action'],
```

Comment:
@amolpati30 why this change in 6.14.z, I see its not present in you 6.15.z and 6.16.z cherypicks?